src/main/resources/api/swagger-config.json

```diff
@@ -7,6 +7,5 @@
   "generateApiTests": false,
   "generateApiDocumentation": false,
   "generateModelTests": false,
-  "generateModelDocumentation": false,
-  "library": "jersey2"
```

Comment:
did this not do something or are we switching to a different library? Maybe easier to be explicit about what library we're using.

Reply:
I tries jersey2 and hope that can solve the oneOf problem, and seems it is not helpful. So no need to have this.
And I guess this wins:
https://github.com/DataBiosphere/terra-resource-janitor/blob/master/build.gradle#L108